### PR TITLE
Increase log level to error on deploy failures

### PR DESCRIPTION
**Pull request by shir-reifenberg**
and remove some dup log

---

_Additional context for reviewer_

---
_Release Notes_: 
None

---
_User Notifications_: 
None

### CODE REVIEW

File: packages/adapter-components/src/client/http_client.ts [hunk 1]
```diff
@@ -263,7 +269,9 @@ export abstract class AdapterHTTPClient<TCredentials, TRateLimitConfig extends C
           responseText,
         )
       } else {
-        log.warn(`failed to ${method} ${url} with error: ${error}, stack: ${error.stack}, ${responseText}`)
+        log.error(
```
Comment (netama) [inline packages/adapter-components/src/client/http_client.ts:272]: I don't think we should increase this to `error` since we might treat these as valid if they're listed as http success codes. what's the purpose of this change?

**Review by omrilit — APPROVED** [review]
🏅

**Comment by netama** [comment]
please don't miss my question, also is there a ticket?

**Comment by shir-reifenberg** [comment]
closing after discussion with @netama
